### DISCUSSION TITLE
Support for Semantic Versioning (error WIX0242)

### DISCUSSION THREAD

**Discussion by vdschuck** [wixtoolset/issues#7551] [Questions]
I wonder if there is a way to incorporate Semantic Versioning in Wix currently? 

When I try to create a release candidate version, I get an error: 

`WIX0242: Invalid product version '0.1.0-rc.0'. MSI product versions must have a major version less than 256, a minor version less than 256, and a build version less than 65536.  The revision value is ignored but version labels and metadata are not allowed.`

I found a page about this in the documentation (https://wixtoolset.org/docs/development/wips/burn-engine-semantic-versioning/). Can anyone confirm if it is up to date? Is it expected to be included in the next release?

**Comment by barnson** [comment]
That spec is about bundles, not MSI packages. MSI sets the rules for MSI versions.
